spec/definitions/SubscriptionSummaryMetrics.yaml

```diff
@@ -0,0 +1,19 @@
+type: object
+properties:
+  currency:
+    description: Currency (three letter ISO 4217 alpha code) (eg USD, EUR)
+    readOnly: true
+    type: string
+  invoicedAmount:
+    description: "Total amount invoiced. Formatted according to the currency, example: $23.02"
+    readOnly: true
+    type: string
+  collectedAmount:
+    description: "Total amount paid. Formatted according to the currency, example: $15.85"
+    readOnly: true
+    type: string
+  invoiceCount:
+    description: Number of issued, past due, or paid invoices
+    readOnly: true
+    type: number
+    format: int
```

Comment:
There is no such a format: https://swagger.io/docs/specification/data-models/data-types/